features/features.go

```diff
@@ -63,6 +64,7 @@ var features = map[FeatureFlag]bool{
 	EnforceOverlappingWildcards: false,
 	OrderReadyStatus:            false,
 	CAAValidationMethods:        false,
+	AllowTLSALPN01Challenge:     false,
```

Comment:
I'm not _entirely_ sure we need a feature flag for this as it can be turned on/off with the `pa.challenges` map in CA/RA configs. Is there something obvious I'm missing here?